ios/Classes/TiMapSnapshotterProxy.h

```diff
@@ -0,0 +1,15 @@
+/**
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2009-2016 by Appcelerator, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+
+#import <UIKit/UIKit.h>
+#import "TiViewProxy.h"
+
+@interface TiMapSnapshotterProxy : TiViewProxy {
```

Comment:
Why is this a view-proxy (`TiViewProxy`) and no proxy (`TiProxy`)? I don't see any view-related operations that require this kind of proxy.